pages/rolodex.js

```diff
@@ -0,0 +1,85 @@
+window.$ = window.jQuery = require('jquery')
+var $ = window.$
+
+module.exports = function () {
+  var itemWidth = $('.rolodex-item').first().width()
+  var animating = false
+
+  $('[data-dex="prev"]').on('click', slide.bind(null, '-='))
+  $('[data-dex="next"]').on('click', slide.bind(null, '+='))
+
+  initialRender()
+
+  function slide (dir, e) {
+    e.preventDefault()
+    if (animating) return
+    manageAnimation(dir)
+    manageActiveClass(dir)
+    manageActiveContent()
+  }
+
+  function manageAnimation (dir) {
+    var notSetUp = ($('.rolodex-item').first().position().left === 0 && dir === '+=')
```

Comment:
Prefer enumerations over more cryptic `-=` hieroglyphs.
It'd be more readable if dir could be one of: `['left', 'right']` or perhaps if we want language neutral symbology `['<--', '-->']`,

That'll require a mapping from the human name to the mechanism by which the direction will be affected, but that's a good thing.